source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -204,7 +205,44 @@ If used to set up ovirt-provider-ovn, engine-setup will perform the following ta
 * install ovs/ovn packages
 * install ovirt-provider-ovn package
 * add and configure a default External Network OVN provider. The engine provider will be configured to connect to provider on localhost. The provider will also be configured with the user and password specified during the setup process.
+* configure the OVN and ovirt-provider-ovn public key infrastructure.
 
+### Configuring PKI manually
+
+The PKI for the provider and driver are configured automatically by engine-setup and vdsm-tool.
+In case where the provider is not installed together with ovirt-engine, PKI will need to be created and configured manually.
+
+In such a case, key/certificate pair signed by a common authority will be needed for the provider, ovn north db, ovn south db, and each of the ovn controllers.
+
+Some of the possible ways to generate key/certificate pairs are using [OpenSSL](https://www.openssl.org/) or openvswitch [ovs-pki command](http://openvswitch.org/support/dist-docs/ovs-pki.8.txt).
+
+The ovn north db must be configured using the following commands:
+
+    ovn-nbctl set-ssl <private key file> <certificate file> <ca certificate file>
+    ovn-nbctl set-connection pssl:6641
+
+The ovn south db must be configured using the following commands:
+
+    ovn-sbctl set-ssl <private key file> <certificate file> <ca certificate file>
+    ovn-sbctl set-connection pssl:6642
+
+The provider must be configured by setting the follwing values in the configuration file (/etc/ovirt-provider-ovn/ovirt-provider-ovn.conf):
+
+    [SSL]
+    ssl_enabled=true # enables/disables https for the REST API
+    key-file=<private key file>
+    cert-file=<certificate file>
+    cacert-file=<ca certificate file>
+
+The PKI of the provider will be used to connect to the OVN north db, as well as for the https connection.
+The CA certificate must used for signing the certificate must be added to the external provider trust store on the engine:
+
+
+    keytool -import -alias <certificate alias> -keystore /var/lib/ovirt-engine/external_keystore -file <ca certificate file> -noprompt -storepass <store password, default: 'changeit'>
```

Comment:
This is the default value. Removing
Replacing with a pointer to a config file which contains both the keystore path and password